test/integration/default/serverspec/server_spec.rb

```diff
@@ -22,3 +22,11 @@
 describe service('nginx') do
   it { should be_running }
 end
+
+describe file('/opt/formsender/source/conf.py') do
+  default = "'default': 'support@osuosl.org'"
+  support = "'support': 'support@osuosl.org'"
+  root    = "'root': 'root@osuosl.org'"
+  it { should exist }
+  its(:content) { should match(/EMAIL = {#{default},#{support},#{root}}/) }
```

Comment:
While this is one way to fix it, I'd tend to prefer sticking to what we've done [in the past](https://github.com/osuosl-cookbooks/osl-nrpe/blob/master/test/integration/default/serverspec/server_spec.rb#L42-L48). Mind taking a look at that link and adjusting it to be like that?